TheMovieDB/Data/MovieDataSource.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  MovieService.swift
+//  TheMovieDB
+//
+//  Created by Intern on 14/07/23.
+//
+
+import Foundation
+
+protocol MovieDataSourceProtocol {
+    func getPopularMovies(page: Int, completion: @escaping (QueryResult?) -> Void)
+}
+
+let AUTH: String = "eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiJmNjNiMTg2NzRmZjlmNzAzYjVlYjU3NjcyYmFmZDljMiIsInN1YiI6IjY0OGFhOTdiMjYzNDYyMDBjYTE4Nzk4YSIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.tQgyjS495o4qqwycv2DYovgi544ubIwy6ZtujaiqWok"
```

Comment:
you can move this to new file, name it Constant.swift 

and you can put it inside enum

`enum Constants {
     static let Auth: String = "dehdehdehdehud"
}`

so if other developer need that auth they just need Constant.Auth